### PR TITLE
Fix incorrect function call

### PR DESCRIPTION
**Pull request by fyellin**
Partial fix for #429.  

"C" wrapper was calling the wrong Rust method.

### CODE REVIEW

**Review by Korijn — APPROVED** [review]

**Comment by Korijn** [comment]
Sorry for that approval, I didn't notice it was another repo, haha!

**Comment by fyellin** [comment]
@almarklein 

Korijn approved this pull by request by mistake.  It will not led me add you as a reviewer.  One line fix for an obvious copy-and-paste error.

**Review by almarklein — APPROVED** [review]
Looks like a typo to me indeed 👍

**Comment by fyellin** [comment]
@almarklein I do not have write access.  I cannot do the merge.

**Comment by almarklein** [comment]
On this repo I usually don't merge before @rajveermalviya has had a look, but this looks rather obvious, so let's merge.

**Pull request merged**
